### PR TITLE
Docs: update app gallery

### PR DESCRIPTION
**Pull request by JakobMiksch**
- add  descriptions to every applications
- add new applications
- link source code if available

@juergenhah @justb4 feel free to edit your examples or add new ones

image

### CODE REVIEW

File: docs/gallery.md [hunk 1]
```diff
@@ -2,32 +2,90 @@
...
 A list of existing Wegue projects.
 
-[Architect Oeds de Leeuw Wieland](https://bevinggevoeld.nl/oeds/groot/
-)
+---
 
-![](_media/screenshot_bevinggevoeld.jpg)
+## [Time-Travel through Rhineland-Palatinate](https://apps.meggsimum.de/wegue-demos/rlp-historische-tk/?year=1960)
 
+This demo shows how to interact with time-dependent geodata. It visualizes topographic maps of Rhineland-Palatinate in Germany from 1885 until 2020. The data is provided by [Landesamt für Vermessung und Geobasisinformation
```
Comment (chrismayer) [inline docs/gallery.md:9]: Maybe better `from 1885 to 2020`?

File: docs/gallery.md [hunk 2]
```diff
@@ -2,32 +2,90 @@
...
+Rheinland-Pfalz (LVermGeo)](https://lvermgeo.rlp.de/de/startseite/) as a time enabled Web Map Service (WMS).
```
Comment (chrismayer) [inline docs/gallery.md:10]:
```suggestion
Rheinland-Pfalz (LVermGeo)](https://lvermgeo.rlp.de/de/startseite/) as a time enabled Web Map Service (WMS-Time).
```

File: docs/gallery.md [hunk 3]
```diff
@@ -2,32 +2,90 @@
...
+---
 
-[Heron](https://wegue.heron-mc.org/)
+## [Countries of the World](https://apps.meggsimum.de/wegue-demos/global/)
 
-![](_media/screenshot_heron.jpg)
+[Source Code](https://github.com/meggsimum/wegue-demos)
 
-[Naturpark Lüneburger Heide](https://map.naturpark-lueneburger-heide.de/)
+A proof of concept how to include data from a global scale. It displays cities, countries and timezones. By clicking a on a location some information will be shown.
```
Comment (chrismayer) [inline docs/gallery.md:28]:
```suggestion
A proof of concept how to include data from a global scale. It displays cities, countries and time zones. By clicking on a location on the map detail information will be shown.
```

File: docs/gallery.md [hunk 4]
```diff
@@ -2,32 +2,90 @@
...
 
-![](_media/screenshot_lueneburg.jpg)
+![screenshot countries of the world](_media/screenshot_global.jpg)
 
-[Defibrillatoren Standorte in Wien](https://gisolutionsat.github.io/wegue)
+---
+
+## [Architect Oeds de Leeuw Wieland](https://bevinggevoeld.nl/oeds/groot/)
+
+A webapplication for exploring the work of the Dutch architect Oeds de Leeuw Wieland. A click on a map icon opens a detailed information card with both data and images.
```
Comment (chrismayer) [inline docs/gallery.md:36]:
```suggestion
A web application for exploring the work of the Dutch architect Oeds de Leeuw Wieland. By clicking on a map icon a card with detailed information is opened showing both data and images. 
```

File: docs/gallery.md [hunk 5]
```diff
@@ -2,32 +2,90 @@
...
+
+A webapplication for exploring the work of the Dutch architect Oeds de Leeuw Wieland. A click on a map icon opens a detailed information card with both data and images. 
+
+![screenshot architect](_media/screenshot_bevinggevoeld.jpg)
+
+---
+
+## [Rain and Flood Hazard Map](https://www.starkregengefahr.de/baden-wuerttemberg/glems/)
+
+An application showing urban flooding events for different regions depending on various environmental scenarios. It is embeded in a Wordpress page und makes use of the `appCtx` parameter in the URL for instantly navigating to the desired region.
```
Comment (chrismayer) [inline docs/gallery.md:44]:
```suggestion
An application showing urban flooding events for different regions depending on various environmental scenarios. It is embeded in a Wordpress page and makes use of the `appCtx` parameter in the URL for instantly navigating to the desired region.
```

File: docs/gallery.md [hunk 6]
```diff
@@ -2,32 +2,90 @@
...
+
+![screenshot rain and flood hazard](_media/screenshot_starkregengefahr.jpg)
+
+---
+
+## [Traffic Analytics - Geomer Here](https://apps.meggsimum.de/geomer-here/)
+
+This application visualises time-dependent traffic data from "here". It makes use of a separate time-slider component.
```
Comment (chrismayer) [inline docs/gallery.md:52]:
```suggestion
This application visualizes time-dependent traffic data from "here". It makes use of a separate time-slider component. The traffic data is loaded as vector tiles.
```

File: docs/gallery.md [hunk 7]
```diff
@@ -2,32 +2,90 @@
...
+
+This example shows how very basic maps can quickly be generated using Wegue. It shows the outlines of two regions combined with some metadata. Maps like this can be made without any coding. 
+
+![screenshot sauber study regions](_media/screenshot_sauber.jpg)
+
+---
+
+## [Heron](https://wegue.heron-mc.org/)
+
+This map displays windmills in the Netherlands. By clicking on a windmill location a sidebar with additional information and a picture opens up.
```
Comment (chrismayer) [inline docs/gallery.md:68]:
```suggestion
This webmap displays windmills in the Netherlands. By clicking on a windmill location a sidebar with additional information and a picture opens up. The application shows how Wegue can be used as a next generation client for [Heron](https://heron-mc.org/) 
```

File: docs/gallery.md [hunk 8]
```diff
@@ -2,32 +2,90 @@
...
+
+![screenshot windmills](_media/screenshot_heron.jpg)
+
+---
+
+## [Nature Park "Lüneburger Heide"](https://map.naturpark-lueneburger-heide.de/)
+
+This website shows a classic Web-GIS application. The user can choose from a variety of different layers from the layer tree. Additional many common functionalities like measure tools, PDF-export, GeoCoding and data download are implemented.
```
Comment (chrismayer) [inline docs/gallery.md:76]:
```suggestion
This website shows a classic Web-GIS application. The user can choose from a variety of different layers listed in a layer tree. Additional many common functionalities like measure tools, PDF-export, GeoCoding and data download are implemented.
```

**Review by chrismayer — COMMENTED** [review]
Thanks for this @JakobMiksch. I added a few suggestions fixing typos and with some alternative wording. Feel free to take whatever you think.

File: docs/gallery.md (hunk 1 above)
Comment (JakobMiksch) [reply docs/gallery.md:9]: I think both `to` and  `until` is  possible

Comment (JakobMiksch) [reply docs/gallery.md:9]: At least that's  what I could quickly find in the internet. That's why I would keep it.

**Comment by JakobMiksch** [comment]
@chrismayer thanks for your review. I applied the suggestion. From my side we are ready to merge.

**Comment by chrismayer** [comment]
Yes, this is good to go IMHO. Thanks @JakobMiksch!

**Pull request merged**